pygenesys/tests/test_inp/test_inputfile.py

```diff
@@ -0,0 +1,24 @@
+"""
```

Comment:
Ok, so this isn't a test file really, it's an example input file? Is that right? Or are you going to add a driver test and use this input file? The convention of using `test_` before a file means that it's an actual test file, not a utility file used by a test. Consider renaming this file to make its purpose clear.

Reply:
yeah that's right. I'll create an issue for this